### DISCUSSION TITLE
Hydrow

### DISCUSSION THREAD

**Discussion by pbcjh10** [JaapvanEkris/openrowingmonitor#53] [Rower Profile Development]
I have a hydrow which is a joy to row on but I'd like to break out of their subscription model 
It's possible to mount a TV where I could stream YouTube rowing or peleton rowing etc 

The connection from rower to screen is just a simple 3pin headphones jack. I wonder if anybody has tried to set this up in ORW or would be interested in trying?

**Comment by JaapvanEkris** [comment]
A beauty of a machine, that is for sure!

In theory it should work if there is some signal indicating flywheel position on the 3.5mm plug. Can you hook up a scope or something similar to see what kind of signal is coming through? It it is simple stream of pulses for each rotation, it is doable (we even tamed the Concept2 RowErg with a sinoid signal). Most rowers just send a simple pulse based on a magnet passing a sensor.

Jaap

PS:have you seen this thread: https://www.reddit.com/r/HydrowCrew/comments/19dsm8h/has_anyone_hacked_their_hydrow_display/

**Reply by pbcjh10** [reply]
Thanks Jaap, I've never owned a scope. Im happy to try if you can talk me through what I need to acquire and how to interpret?

**Reply by JaapvanEkris** [reply]
I think @Abasz has the most experience with the scope here. Alternatively, you add magnets and a new sensor to the setup, which prabbaly is a lot easier.

**Reply by Abasz** [reply]
The 3 pin suggests a hall sensor, one is Vcc, one is Gnd, and one is Signal. Now which is which that is difficult. The sensor needs power to measure anything on the signal pin and also halls generally need more than 3.3v. 

I suggest that you get similar jack cable, cut it and plug it into the monitor, then you measure the voltage with a multimeter (not need to be fancy, just basic) on the wires. from that you could determine which sleeve on the jack is what. 

Look for ground and vcc first. Vcc should have a stable voltage on it. Its possible that two will have the same voltage. That means one is signal which has a pullup resisitor, and the other is vcc. but the important part is that you need to know the voltage.

Then get a battery or a power supply able to produce that voltage and you can try playing with the jack of the sensor. You trigger it with a magnet, the signal will be high (same or nearly the same as supply) when no magnet it will be low (Gnd). The one that changes is the signal.

**Comment by DXCanas** [comment]
Been thinking about this. Local listings are packed with giant paperweights shaped like fancy rowing machines from folks who no longer want the sub. I’ve got access to one myself. 

What I’ve been thinking about, though, is that the giant display also has a resistance adjustment toggle. So I wonder how that might work. Maybe one of the 3 pins would act as input?

**Reply by JaapvanEkris** [reply]
> What I’ve been thinking about, though, is that the giant display also has a resistance adjustment toggle. So I wonder how that might work. Maybe one of the 3 pins would act as input?

As far as I know, it actively controls the magnetic resistance at 240Hz. I don't know what happens if you disconnect that signal: it might get a magnet stuck in one setting, or it might remove the resistance altogether. In general magnetic resistance is far from optimal as an air/water braked flywheel adapts much better to the activated muscles in the rowing stroke. From a technical perspective, a magnetic rower has a static speed independent drag**force**, where an air/water braked flywheel has a speed dependent drag**force**. Hydrow got around this by using electronics to simulate the speed dependence: they measure the flywheel speed and adjust the magnet's force accordingly. It should be a high-frequency circuit, as they use 240Hz to do these corrections. If that is inside the monitor, it might become a useless machine.
